### DISCUSSION TITLE
Im still new so if everyone could be a patient and understanding and if you should want to get me some insights of help would be much appreciated thank you for understanding

### DISCUSSION THREAD

**Discussion by ghost** [community/community#62186] [New to GitHub]
### Select Topic Area

Question

### Body

Im still new so if everyone could be a patient and understanding and if you should want to get me some insights of help would be much appreciated thank you for understanding

**Comment by GitStoopID** [comment]
Wecome, same as well.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Lileazzy, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your question entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. 

Check out this [post](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
